src/main/java/calculator/StringSumCalculator.java

```diff
@@ -26,13 +28,10 @@ public static int sum(int[] intNumbers) {
     }
 
     private static String[] splitNumbersWithDelimiter(String numbers) {
-        final String FRONT_OF_CUSTOM_DELIMITER = "//";
-        final String END_OF_CUSTOM_DELIMITER = "\n";
-
-        if (numbers.matches(CUSTOM_DELIMTER_REGEX)) {
-            String[] delimiterAndNumbers = numbers.split(END_OF_CUSTOM_DELIMITER);
-            String customDelimiter = delimiterAndNumbers[0].substring(FRONT_OF_CUSTOM_DELIMITER.length());
-            return delimiterAndNumbers[1].split(customDelimiter);
+        Matcher matcher = Pattern.compile(CUSTOM_DELIMITER_REGEX).matcher(numbers);
```

Comment:
피드백 반영 👍 
정규식 문자열뿐만 아니라 Pattern 객체도 상수로 관리해 보셔도 좋을 것 같아요
```suggestion
        Matcher matcher = CUSTOM_DELIMITER_PATTERN.matcher(numbers);
```

Reply:
원시값에 대해서만 상수화시켰는데, 객체도 고려할 수 있겠군요! 👍